layouts/default.vue

```diff
@@ -2,7 +2,7 @@
   <div class="bg-gray-900">
     <global-header />
 
-    <main>
+    <main style="min-height: calc(100vh - 198px)">
```

Comment:
Can it not just use `min-h-screen`?

Reply:
That was my first thought but unfortunately that didn't work.

Reply:
I'll have a play. Where are you getting `198px` from?